modules/schematics/src/component-store/index.spec.ts

```diff
@@ -148,24 +138,22 @@ describe('component-store', () => {
     );
     const content = tree.readContent(`${projectPath}/src/app/app.component.ts`);
 
-    expect(content).toMatch(/FooStore/);
+    expect(content).toMatch(/FooStore/i);
+    expect(content).toMatchSnapshot();
   });
 
   it('should fail if specified component does not exist', async () => {
     const options = {
       ...defaultOptions,
       component: `${projectPath}/src/app/app.componentXXX.ts`,
     };
-    let thrownError: Error | null = null;
-    try {
-      await schematicRunner.runSchematic('component-store', options, appTree);
-    } catch (err: any) {
-      thrownError = err;
-    }
-    expect(thrownError).toBeDefined();
+    const thrownError: Error | null = null;
```

Comment:
It seems that `thrownError` constant is unused after refactoring

```suggestion
```

Reply:
Thanks Marko :)